Code/DistGeom/DistGeomUtils.h

```diff
@@ -148,6 +148,22 @@ RDKIT_DISTGEOMETRY_EXPORT ForceFields::ForceField *construct3DForceField(
 
   <b>NOTE:</b> the caller is responsible for deleting this force field.
 
+*/
```

Comment:
You probably need to update the comment to explain what the extra argument is, right?

Reply:
done